app/containers/Avatar.js

```diff
@@ -40,7 +40,7 @@ export default class Avatar extends PureComponent {
 		const room = type === 'd' ? text : `@${ text }`;
 		// Avoid requesting several sizes by having only two sizes on cache
 		const uriSize = size === 100 ? 100 : 50;
-		const uri = avatar || `${ baseUrl }/avatar/${ room }?format=png&width=${ uriSize }&height=${ uriSize }`;
+		const uri = avatar || `${ baseUrl }/avatar/${ room }?format=png&width=${ uriSize }&height=${ uriSize }&ts=${ timeStamp }`;
```

Comment:
Adding `ts` to the url will break the cache and it's not an option.
Your solution will make the app download the same avatars over and over.

Reply:
I know. But the only way the image will update once is fetching from the url is that url change. I tried to change in the `FastImage` the way cache works using `FastImage.cacheControl.web` but I got the same result. No update.

Reply:
Any other luck?